plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -1063,33 +1062,7 @@ private boolean isValidPartition(Table table, String partitionName)
             List<String> columnNames,
             TupleDomain<String> partitionKeysFilter)
     {
-        if (partitionKeysFilter.isNone()) {
```

Comment:
Is `getPartitionNamesByFilter` best effort now? I don't think `SemiTransactionalHiveMetastore#doGetPartitionNames` works like that (filters returner results), does it?

Reply:
It is best effort. The only caller is `HivePartitionManager#getPartitions` which does the filtering.